.github/workflows/build-test.yml

```diff
@@ -1061,6 +1060,8 @@ jobs:
         with:
           node-version: ${{env.NODE_VERSION}}
 
+      - run: npm i -g vsce
```

Comment:
Should we just add this to our dev dependencies?

Reply:
I was about to ask, I thought that it already was. Not sure how build would have worked without this.

Reply:
Not sure, I kinda feel npm ci would take longer.
I'd prefer to leave it out for now